src/buza/admin.py

```diff
@@ -6,3 +6,40 @@
 @admin.register(models.User)
 class UserAdmin(admin.ModelAdmin):
     list_display = ['username', 'school', 'grade']
+
+
+class AnswerInline(admin.TabularInline):
+    extra = 0
+    model = models.Answer
+
+    fields = ['author', 'body']
+    raw_id_fields = ['author']
+
+
+@admin.register(models.Question)
+class QuestionAdmin(admin.ModelAdmin):
+    date_hierarchy = 'created'
```

Comment:
how do we allow admin to choose if they want to view months or days, to prevent this from getting cluttered?

Reply:
The date hierarchy view is actually pretty intelligent about this: it does aggregate queries on the actual data to see what years, months, and days are represented, so it only shows the ones that have values, and lets you drill up and down, without much clutter. :)